composer.json

```diff
@@ -6,11 +6,11 @@
     "homepage": "https://github.com/twilio/authy-php",
     "license": "MIT",
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "~6.0"
+        "php": "^7.2.5",
+        "guzzlehttp/guzzle": "~7.0"
```

Comment:
Maybe it should be `"~6.0||~7.0"` here?

Reply:
:+1: Then it can be released as next minor version.

Reply:
Updated. Thanks for the feedback!

Reply:
Oops, that should've been `"~5.0||~7.0"`...